### PR TITLE
Solving issues

### PR DESCRIPTION
**Pull request by mihaelavalac**

### CODE REVIEW

File: resources/views/layouts/navigation.blade.php [hunk 1]
```diff
@@ -41,43 +41,42 @@
...
+                    <x-slot name="content">
+                        <!-- Authentication -->
+                        <form method="POST" action="{{ route('logout') }}">
+                            @csrf
+                            <x-dropdown-link :href="route('profile')">
+                                {{ __('Profile') }}
+                            </x-dropdown-link>
+                            <x-dropdown-link :href="route('wishlist')">
+                                {{ __('My Wishlist') }}
+                            </x-dropdown-link>
```
Comment (ryvrook) [inline resources/views/layouts/navigation.blade.php:66]: Moved Wishlist

File: resources/views/components/dropdown.blade.php [hunk 2]
```diff
@@ -28,10 +28,6 @@
 
     <div x-show="open" x-transition:enter="transition ease-out duration-200" x-transition:enter-start="transform opacity-0 scale-95" x-transition:enter-end="transform opacity-100 scale-100" x-transition:leave="transition ease-in duration-75" x-transition:leave-start="transform opacity-100 scale-100" x-transition:leave-end="transform opacity-0 scale-95" class="absolute z-50 mt-2 {{ $width }} rounded-md shadow-lg {{ $alignmentClasses }}" style="display: none;" @click="open = false">
         <div class="rounded-md ring-1 ring-black ring-opacity-5 ">
-        <x-responsive-nav-link  :href="route('wishlist')">
-            {{ __('My Wishlist') }}
-        </x-responsive-nav-link>
-
```
Comment (ryvrook) [inline resources/views/components/dropdown.blade.php:34]: Removed improper location. Good

File: resources/views/layouts/navigation.blade.php [hunk 3]
```diff
@@ -101,54 +100,49 @@
...
+                    <!-- Authentication -->
+                    <x-responsive-nav-link :href="route('games')" :active="request()->routeIs('games')">
+                        {{ __('Game List') }}
+                    </x-responsive-nav-link>
+                    <x-responsive-nav-link :href="route('wishlist')">
+                        {{ __('My Wishlist') }}
+                    </x-responsive-nav-link>
+                    <x-responsive-nav-link :href="route('profile')" :active="request()->routeIs('profile')">
+                        {{ __('Profile') }}
+                    </x-responsive-nav-link>
```
Comment (ryvrook) [inline resources/views/layouts/navigation.blade.php:127]: Proper addition

File: resources/views/layouts/navigation.blade.php [hunk 4]
```diff
@@ -101,54 +100,49 @@
...
-                <!-- Authentication -->
-                <x-responsive-nav-link :href="route('wishlist')">
-                    {{ __('My Wishlist') }}
-                </x-responsive-nav-link>
-                <form method="POST" action="{{ route('logout') }}">
-                    @csrf
-                    <x-responsive-nav-link :href="route('logout')" onclick="event.preventDefault();
+                    <form method="POST" action="{{ route('logout') }}">
+                        @csrf
+                        <x-responsive-nav-link :href="route('logout')" onclick="event.preventDefault();
```
Comment (ryvrook) [inline resources/views/layouts/navigation.blade.php:131]: Added Logout

File: resources/views/profile.blade.php [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <x-app-layout>
     <x-slot name="header">
         <h2 class="font-semibold text-xl text-gray-800 leading-tight">
-            {{ __('profile') }}
+            {{ __('My Profile') }}
```
Comment (ryvrook) [inline resources/views/profile.blade.php:4]: Rename

File: routes/web.php [hunk 6]
```diff
@@ -21,7 +21,7 @@
 // });
 
 Route::get('/', function () {
-    return view('dashboard');
+    return view('profile');
```
Comment (ryvrook) [inline routes/web.php:24]: REname

**Review by ryvrook — APPROVED** [review]
Good additions from what I can see.

**Review by beemteam2 — APPROVED** [review]
thanks to both of yall for working on this and getting it straightened out.

**Pull request merged**